### PR TITLE
feat(extentions): Put Bitbucket in global integrations

### PR DESCRIPTION
**Pull request by lauryndbrown**
- Puts Bitbucket in the list of global integrations
- Creates the first step of setup, where a descriptor file is sent to bitbucket.org
- Does not yet do anything with that information Will continue with another pull request

### CODE REVIEW

File: src/sentry/integrations/bitbucket/descriptor.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+class BitBucketDescriptorEndpoint(Endpoint):
+    authentication_classes = ()
+    permission_classes = ()
+
+    def get(self, request):
+        return self.respond(
+            {
+                'key': 'sentry-bitbucket',
+                'name': 'Sentry Bitbucket',
```
Comment (macqueen) [inline src/sentry/integrations/bitbucket/descriptor.py]: i'd make this `Sentry for BitBucket`

File: src/sentry/integrations/bitbucket/descriptor.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+                'description': 'A sentry integration',
```
Comment (macqueen) [inline src/sentry/integrations/bitbucket/descriptor.py]: capitalize Sentry

File: src/sentry/integrations/bitbucket/integration.py [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+from sentry.pipeline import NestedPipelineView
+from sentry.identity.pipeline import IdentityProviderPipeline
+from django.utils.translation import ugettext_lazy as _
+DESCRIPTION = """
+BitBucket for Sentry.io
+"""
+metadata = IntegrationMetadata(
+    description=DESCRIPTION.strip(),
+    author='The Sentry Team',
+    noun=_('Repository'),
```
Comment (macqueen) [inline src/sentry/integrations/bitbucket/integration.py]: do you have to install on each repository? or should this be something that refers to a bitbucket org or account?

File: src/sentry/conf/server.py [hunk 4]
```diff
@@ -1236,13 +1236,15 @@ def create_partitioned_queues(name):
 SENTRY_USE_X_FORWARDED_FOR = True
 
 SENTRY_DEFAULT_INTEGRATIONS = (
+    'sentry.integrations.bitbucket.BitBucketIntegrationProvider',
```
Comment (macqueen) [inline src/sentry/conf/server.py]: maybe just comment out this line until it's fully functional? i don't think we need to actually have this in the list if it's just going to blow up at some point in the pipeline if an internal user tries to add it.

File: src/sentry/integrations/bitbucket/integration.py (hunk 3 above)
Comment (lauryndbrown) [reply src/sentry/integrations/bitbucket/integration.py]: @macqueen Bitbucket doesn't appear to have organizations. You have repositories, teams, and users. But many teams can access the same repositories; it's not a 1-to-1. I can do the bitbucket.org account instead.

Comment (lauryndbrown) [reply src/sentry/integrations/bitbucket/integration.py]: Here's a list to the api reference I'm looking at. https://developer.atlassian.com/bitbucket/api/2/reference/resource/users let me know if I'm missing something

**Comment by lauryndbrown** [comment]
Closing this PR. Will follow up with a different PR

**Pull request closed without merging**